packages/terra-site/src/examples/heading/HeadingSizes.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Like Emily suggested earlier, let's include a few headers that mix the two together. I don't think we need all of the possible combinations, but enough to show that the two can be mixed together.